api/address/routes.go

```diff
@@ -28,9 +30,17 @@ type accountResponse struct {
 }
 
 // Routes defines address related routes
-func Routes(router *gin.RouterGroup) {
-	router.GET("/:address", GetAccount)
-	router.GET("/:address/balance", GetBalance)
+func Routes(router *gin.RouterGroup, routesConfig config.ApiRoutesConfig) {
+	addresRoutes, ok := routesConfig.APIPackages["address"]
+	if !ok {
+		return
+	}
+	if configparser.CheckEndpoint(":address", addresRoutes) {
```

Comment:
`CheckEndpoint` doesn't say much about what it's going to check. `IsEndpointActive` maybe?

Reply:
done